cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/velocity/listener/VelocityPlayerListener.java

```diff
@@ -72,8 +72,14 @@ public void handle(ServerConnectedEvent event) {
 
     @Subscribe
     public void handle(KickedFromServerEvent event) {
-        VelocityCloudNetHelper.getNextFallback(event.getPlayer())
+        if(!event.getPlayer().isActive())
+            return;
+
+        BridgeProxyHelper.handleConnectionFailed(event.getPlayer().getUniqueId(), event.getServer().getServerInfo().getName());
+
```

Comment:
No need for the empty line

Reply:
What empty line? The first or the second?

Since I did some changes now could you review them too? Is it fine now?

Reply:
Looks good now